spark-publish/index.js

```diff
@@ -49,7 +49,7 @@ export const publish = async ({
   // Share measurements
   let start = new Date()
   const file = new File(
-    [JSON.stringify(measurements)],
+    [measurements.map(m => JSON.stringify(m)).join('\n')],
     'measurements.json',
```

Comment:
```suggestion
    'measurements.ndjson',
```

This could also make it easier to detect old (JSON) vs new (NDJSON) format, if we ever need it.

Reply:
Ah, I see you choose a different approach to support both JSON & NDJSON.

I still think we should use a different file extension, because the file we produce is not a valid JSON.